benchmark_runner/workloads/bootstorm_vm.py

```diff
@@ -66,6 +66,20 @@ def _set_bootstorm_vm_start_time(self, vm_name: str = ''):
         """
         self._bootstorm_start_time[vm_name] = time.time()
 
+    @logger_time_stamp
+    def _wait_virtctl_vm(self, vm_name: str):
+        """
+        This method verifies virtctl into VM and return vm node in success or False if failed
+        @return:
+        """
+        # wait till vm virtctl login
+        if self._oc.get_vm_node(vm_name=vm_name):
+            vm_node = self._oc.get_vm_node(vm_name=vm_name)
+            if self._oc.wait_for_vm_virtctl(vm_name=vm_name):
+                logger.info(f"Successfully virtctl into VM: '{vm_name}' in node {vm_node} ")
+            return vm_node
+        return False
+
```

Comment:
Again, name it something generic so that the underlying mechanism isn't exposed and you can change it in the future without having to change anything outside of `oc.py`.

Reply:
I dont think we should keep the 2 approaches, I plan to shift all to virtctl.

Reply:
We don't need to keep both approaches.  We should simply use one name that's visible outside of the implementation.  You should not have to change any code outside of this file.

If there are more places that need to be changed, please make all of the necessary changes in one PR.

Reply:
I see that we already use wait_for_vm_status if it stop/ready/run.
Do u think we should use the same term for virtctl ?